[DTM] Asset location (#1450)
SoftwareAG/c8y-docs: content/dtm/asset-hierarchy-bundle/creating-assets-ui.md

6. Under **Asset properties**, fill in the required information. Here you find all [asset properties](/dtm/asset-types/#asset-properties) assigned to the asset model that you create.

> For an asset with a location property click **Choose on Map** in the section to set the values for latitude and longitude using map view. Click the full screen icon at the top right corner of the map to view it in full screen.

Author: Modified

> For an asset with a location property, click **Choose on Map** in the section to set the values for latitude and longitude using map view. Click the full screen icon at the top right corner of the map to view it in full screen.

Reviewer: ```suggestion
For an asset with a **Location** property, click **Choose on Map** in the section to set the values for the latitude and longitude using map view. Click the full screen icon at the top right corner of the map to view it in full screen.
```

Author: Modified

> The marker is positioned at the default value set in the location property. If no default value is set, the marker is positioned at D&uuml;sseldorf, Germany. Click or drag the marker to the preferred position to set the value for latitude and longitude. The marker on the map is positioned at an approximated location when either latitude or longitude value is missing. To close the map view, click **Close** above the map.

Reviewer: ```suggestion
The marker is positioned at the default value set in the **Location** property. If no default value is set, the marker is positioned at Duesseldorf, Germany. Click or drag the marker to the preferred position to set the value for both latitude and longitude. The marker on the map is positioned at an approximated location when either the latitude or the longitude value is missing.
```

Author: @MWindel  is it necessary to add Umlaut for Duesseldorf?

Reviewer: @Vagdevi1998 If you spell it "ue" is it fine.

> The marker is positioned at the default value set in the location property. If no defualt value is set,the marker is positioned at D&uuml;sseldorf, Germany. Click or drag the marker to the preferred position to select the value for latitude and longitude. The marker on the map is positioned at an approximated location when either latitude or longitude value is missing. To close the map view, click **close** above the map.

Author: Modified

{{< c8y-admon-info>}}

> The values of latitude and longitude are automatically updated whenever a new location is selected on the map and vice-versa. The Altitude value is not represented on the map.<br>

Reviewer: ```suggestion
The values of latitude and longitude are automatically updated whenever a new location is selected on the map and vice-versa. The altitude value is not represented on the map.<br>
```

Author: Modified

You can view the newly created asset hierarchy in the **Assets** page.
Also see [viewing assets](/dtm/asset-hierarchy/#viewing-assets).

> In case of bulk operation failure, the entire operation is rolled back and user will be navigated to main assets page to cleanup the entities created and try the operation again.

Author: Modified
